alertmanager/alertmanager.yml

```diff
@@ -1,13 +1,19 @@
-route:
-  receiver: 'mail'
-  repeat_interval: 4h
-  group_by: [ alertname ]
+global:
+  resolve_timeout: 1m
 
+route:
+  group_by: ['alertname']
+  group_wait: 10s
+  group_interval: 10s
+  repeat_interval: 1m
+  receiver: 'email'
+  
```

Comment:
I'd only add some explanatory comments for these, something like:
# Alerts grouped by the 'alertname' label(s). When a new group of alerts is created, wait for 10s before firing the alerts. After the alerts were fired, wait for 10s before sending another alerting batch for the group. Wait for 1m overall before repeating.